css_01/css_01.css

```diff
@@ -0,0 +1,49 @@
+#bodyContent
```

Comment:
Usually most of the time classes are used for css styling and not ids. It is not wrong the way it is done currently, I am requesting no change, just giving more information on the topic. Classes are usually easier to be reused. You may want to read more here: https://stackoverflow.com/a/12889421/3837638